build.sbt

```diff
@@ -19,7 +19,7 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 // because these versions switched to the same Java collection wrappers used in Scala 2.13.
 // These wrappers lack dedicated Kryo serializers, which we added in flink-scala-2.13 library https://github.com/TouK/flink-scala-2.13.
 // To bump Scala 2.12 we would need to do flink-scala-2.12 similar to flink-scala-2.13.
-val scala212 = "2.12.10"
+val scala212 = "2.12.15"
```

Comment:
2.12.20 and 2.13.16? We need to update flink-scala anyway, so let's use this opportunity to update to the newest versions

Reply:
fixed